edi_stock_oca/models/stock_picking.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 OCA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class StockPicking(models.Model):
+    _name = "stock.picking"
+
+    _inherit = ["stock.picking", "edi.exchange.consumer.mixin"]
+
+    disable_edi_auto = fields.Boolean(
```

Comment:
Question: would be interesting to have this on the consumer mixin?
Also, matter of taste maybe but I'd prefer to prefix such fields w/ `edi`. In this case I vote for `edi_auto_disabled`.

Reply:
Well, the field can be added on edi.exchange.mixin from edi_oca, but, at the end, we will allways need to inherit on the models in order to set the states and so on :thinking: 

I agree with the change of name and move to `edi`, but it is already being used from l10n-spain facturae (at least...) could we add a comment to rename it on 16?